source/common/http/alternate_protocols_cache_manager_impl.cc

```diff
@@ -53,6 +53,20 @@ AlternateProtocolsCacheSharedPtr AlternateProtocolsCacheManagerImpl::getCache(
 
   AlternateProtocolsCacheSharedPtr new_cache = std::make_shared<AlternateProtocolsCacheImpl>(
       dispatcher.timeSource(), std::move(store), options.max_entries().value());
+
+  for (const envoy::config::core::v3::AlternateProtocolsCacheEntry& entry :
+       options.prepopulated_entries()) {
+    const AlternateProtocolsCacheImpl::Origin origin = {"https", entry.hostname(), entry.port()};
+    std::vector<AlternateProtocolsCacheImpl::AlternateProtocol> protocol = {
+        {"h3", entry.hostname(), entry.port(),
+         dispatcher.timeSource().monotonicTime() + std::chrono::hours(168)}};
+    OptRef<const std::vector<AlternateProtocolsCacheImpl::AlternateProtocol>> existing_protocols =
+        new_cache->findAlternatives(origin);
+    if (!existing_protocols.has_value()) {
```

Comment:
Should we do any sort of validation that we don't already have an entry for |origin|? (or just log in this case?)

Reply:
we can have an entry - as documented, if it's in cache we'll preserve the cache entry over the hard-coded entry.  I didn't think that was worth logging since it should be the common case and would be kinda spammy